fiat-rust/Cargo.toml

```
@@ -1,8 +1,13 @@
 [package]
 name = "fiat-rust"
 version = "0.1.0"
-authors = ["Zoe Paraskevopoulou <zoopar@fb.com>"]
+authors = ["Fiat library authors <jgross@mit.edu>"]
 edition = "2018"
+description = "Fiat-crypto generated Rust for curve25519_64"
+homepage = "http://plv.csail.mit.edu/fiat/"
```

```suggestion
homepage = "https://github.com/mit-plv/fiat-crypto"
```
Fiat is a different project, and we don't actually have a better homepage than the github, yet.